outputs.tf

```diff
@@ -0,0 +1,24 @@
+output "alb_id" {
```

Comment:
Put all outputs available from Terraform along with description which comes from documentation page.

Reply:
Not sure what's meant by this. All outputs have descriptions and values. Can you clarify?